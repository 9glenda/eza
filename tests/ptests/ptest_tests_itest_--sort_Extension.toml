bin.name = "eza"
args = "tests/itest --sort Extension"