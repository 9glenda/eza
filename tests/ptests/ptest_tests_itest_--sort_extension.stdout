a
b
c
d
e
exa
f
g
h
i
j
k
l
m
n
o
p
q
vagrant
image.jpg.img.c.rs.log.png
index.svg
